caffeine/src/main/scala/akka/caffeine/Aggregator.scala

```diff
@@ -0,0 +1,58 @@
+/*
+ * Copyright (C) 2016 Lightbend Inc. <http://www.lightbend.com>
+ */
+package akka.caffeine
+
+import scala.concurrent.duration.FiniteDuration
+
+/**
+ * Aggregation contract
+ *
+ * @tparam K key in the cache
+ * @tparam V value in the cache
+ * @tparam R aggregation type
+ */
+trait Aggregator[K, V, R] {
+
+  /**
+   * Initiate a fresh aggregate.
+   *
+   * @param key
```

Comment:
Either provide nonempty comments or get rid of these (possibly autogenerated) tags.